holidays/countries/british_virgin_islands.py

```diff
@@ -0,0 +1,179 @@
+
+    * [2021] <https://bvi.org.uk/2021-public-holidays/>
+    * [2022] <http://www.bvi.gov.vg/media-centre/revised-2022-public-holidays>
+    * [2023] <https://bvi.gov.vg/media-centre/updatedofficialholidays20231>
+    * [2024] <https://bvi.gov.vg/media-centre/cabinet-approves-2024-public-holidays>
+    * [2025] <https://bvi.gov.vg/media-centre/2025-holiday-calendar-flyer-01>
+    """
+
+    country = "VG"
+    default_language = "en_VG"
+    # %s (observed).
+    observed_label = tr("%s (observed)")
+    subdivisions = ()
+    supported_languages = (
+        "en_US",
+        "en_VG",
+    )
+    start_year = 1967
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        StaticHolidays.__init__(self, BritishVirginIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_TO_PREV_FRI + SUN_TO_NEXT_MON)
+        kwargs.setdefault("observed_since", 2000)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("New Year's Day"))
+        if self._year not in {2004, 2010}:
+            self._add_observed(dt, rule=SAT_SUN_TO_NEXT_MON)
+
+        # Lavity Stoutt's Birthday.
+        self._add_observed(
+            self._add_holiday_mar_7(tr("Lavity Stoutt's Birthday")), rule=ALL_TO_PREV_MON
+        )
+
+        # Commonwealth Day.
+        self._add_holiday_2nd_mon_of_mar(tr("Commonwealth Day"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Easter Monday"))
+
+        # Whit Monday.
+        self._add_whit_monday(tr("Whit Monday"))
+
+        # Sovereign's Birthday.
+        name = tr("Sovereign's Birthday")
+        if self._year < 2020:
+            self._add_holiday_2nd_sat_of_jun(name)
+        else:
+            self._add_holiday_2nd_fri_of_jun(name)
+
+        territory_day_dates = {
+            2015: (JUN, 29),
+            2020: (JUN, 29),
+        }
+        name = (
+            # Colony Day.
+            tr("Colony Day")
+            if self._year < 1978
+            # Territory Day.
+            else tr("Territory Day")
+            if self._year < 2021
+            # Virgin Islands Day.
+            else tr("Virgin Islands Day")
+        )
+        if self._year in territory_day_dates:
+            self._add_holiday(name, territory_day_dates[self._year])
+        else:
+            dt = self._add_holiday_jul_1(name)
+            if self._year not in {2006, 2017}:
+                self._add_observed(dt)
+
+        # Emancipation Monday.
+        emancipation_monday = self._add_holiday_1st_mon_of_aug(tr("Emancipation Monday"))
+
+        # Emancipation Tuesday.
+        self._add_holiday(tr("Emancipation Tuesday"), emancipation_monday + timedelta(days=1))
+
+        # Emancipation Wednesday.
+        self._add_holiday(tr("Emancipation Wednesday"), emancipation_monday + timedelta(days=2))
```

Comment:
```suggestion
        self._add_holiday_1st_mon_of_aug(tr("Emancipation Monday"))

        # Emancipation Tuesday.
        self._add_holiday_1_day_past_1st_mon_of_aug(tr("Emancipation Tuesday"))

        # Emancipation Wednesday.
        self._add_holiday_2_days_past_1st_mon_of_aug(tr("Emancipation Wednesday"))
```
According to Wiki, these holidays called Festival Monday/Tuesday/Wednesday until 2020.